libgadget/forcetree.h

```diff
@@ -20,10 +22,19 @@ struct NODE
         unsigned int DependsOnLocalMass :1;  /* Intersects with local mass */
         unsigned int MixedSofteningsInNode:1;  /* Softening is mixed, need to open the node */
         unsigned int NodeIsDirty :1; /*Node is a toplevel node containing local mass, and its moments need updating*/
+        unsigned int InternalNode :1; /* If true, node contains only other nodes.
```

Comment:
Do we already use the word Internal for something else? Is there chance for confusion?

Reply:
We have InternalTopNode - this might be confusing. It was my original intention a few hours ago to merge them, but that isn't going to work, I think. But I can't think of a better name, do you have any ideas?

Reply:
How about a two-bit bitfield called ChildType which can be enum ChildType{PARTICLE_TYPE=0, NODE_TYPE = 1, PSEUDO_TYPE = 2} ? That way we can avoid some of the messing about with pseudo particle checks as well. The enums can be defines if enum bitfields are not allowed.